backend/src/main/kotlin/blue/mild/covid/vaxx/dto/PatientRegistrationDto.kt

```diff
@@ -0,0 +1,11 @@
+package blue.mild.covid.vaxx.dto
+
+
+data class PatientRegistrationDto(
+    val firstName: String,
+    val lastName: String,
+    val personalNumber: String,
+    val phoneNumber: String,
+    val email: String,
+    val answers: List<AnswerDto>
```

Comment:
A jak jsi rikal, ze tady cches z FE vratit i to potvrzeni, ze souhlasis s GDPR, atd?

Reply:
A zavedl bych syntaxi DtoIn a DtoOut

Reply:
jo, souhlas musime taky, pridam

`DtoIn` a `DtoOut` se mi libi, pridam

Reply:
a @marazt kdyz mam `AnswerDto` ktere jde jak dovnitr tak ven?